src/bidiMapper/domains/context/CdpTarget.ts

```diff
@@ -173,6 +174,27 @@ export class CdpTarget {
     });
   }
 
+  async toggleNetworkIfNeeded(): Promise<void> {
+    const enableNetwork =
+      this.#eventManager.subscriptionManager.isSubscribedToModule(
+        this.#id,
```

Comment:
We imply the fact there is always a BrowsingContext with the very same `#id`. Maybe move the `this.#eventManager.subscriptionManager.isSubscribedToModule` to `BrowsingContext.toggleDomainsIfNeeded()`, and call `cdpTarget.toggleNetwork(enable: bool)` only when needed?

Reply:
Sound OK.